01_slides/05_strings.ipynb

```diff
@@ -609,6 +613,256 @@
       "source": [
         "'JavaScript' in job_qualifications"
       ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "327e7b49",
+      "metadata": {},
+      "source": [
+        "## Strings and functions\n",
+        "\n",
+        "String data works with some of the functions we've seen before, like `print()`, as well as with built-in functions like `len()`, which tells us how long the argument we pass in is."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "0556df8e",
+      "metadata": {},
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "1 fish, 2 fish\n"
+          ]
+        }
+      ],
+      "source": [
+        "# We can print strings, ints, and floats together\n",
+        "print(1, 'fish,', 2, 'fish')"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "6a7903e2",
+      "metadata": {},
+      "outputs": [
+        {
+          "data": {
+            "text/plain": [
+              "80"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        }
+      ],
+      "source": [
+        "len('Sometimes we type long-winded sentences to meet word counts or character counts.')"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "659dd1a9",
+      "metadata": {},
+      "source": [
+        "## String methods\n",
```

Comment:
Thoughts on introducing this concept in the Object-Oriented programming lesson?

Reply:
As discussed on Slack, the string method content will be kept in this lesson to (1) provide additional insights into string manipulation and (2) allow students to use the methods introduced to complete Assignment 1.